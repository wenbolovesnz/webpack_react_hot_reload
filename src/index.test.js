import React from "react";
import { configure, shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

import App from "./app";

configure({ adapter: new Adapter() });
describe("<App /> shallow rendering", () => {
  it("h1 contains correct text", () => {
    const wrapper = shallow(<App />);
    expect(wrapper.find("h1").text()).toBe("Webpack 4 with react starter!");
  });
});
