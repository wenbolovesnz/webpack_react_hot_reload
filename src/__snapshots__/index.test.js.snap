// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> shallow rendering matches the snapshot 1`] = `
<div>
  <h1>
    Webpack 4 with react starter!
  </h1>
  <ul>
    <li>
      <Link
        address="reactjs.orgu"
        text="React"
      />
    </li>
    <li>
      Webpack
    </li>
    <li>
      Jest and Enzyme
    </li>
  </ul>
  <br />
  <button
    onClick={[Function]}
  >
    Like it!
  </button>
</div>
`;
